contrib/kube-prometheus/README.md

```diff
@@ -95,6 +95,8 @@ jsonnet -J vendor -m manifests ${1-example.jsonnet} | xargs -I{} sh -c 'cat $1 |
 
 This script reads each key of the generated json and uses that as the file name, and writes the value of that key to that file.
 
+> You can run also run this script executing the command `make generate-raw` from kube-prometheus base directory of this repository but it is recommended the above option so that you run it in your own infrastructure repository.
```

Comment:
`s/You can run also run/You can also run`

`s/it is recommended the above option/the above option is recommended`